lib/mv_opentelemetry/ecto.ex

```diff
@@ -73,7 +73,7 @@ defmodule MvOpentelemetry.Ecto do
     span_name = name(config, event, source)
     span_opts = %{start_time: start_time, attributes: all_attributes, kind: :client}
 
-    parent_context = OpentelemetryProcessPropagator.fetch_parent_ctx(1, :"$callers")
+    parent_context = OpentelemetryProcessPropagator.fetch_parent_ctx(4, :"$callers")
```

Comment:
<img width="1312" alt="Screenshot 2022-04-11 at 12 21 31" src="https://user-images.githubusercontent.com/2331765/162708270-fe838ed2-feb8-4734-912f-e361dfd58366.png">

This is the best I can do...